Index: list.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/list.txt b/list.txt
new file mode 100644
--- /dev/null	
+++ b/list.txt	
@@ -0,0 +1,1 @@
+Борисов Дмитрий Алексеевич
\ No newline at end of file
Index: YourNameSchool.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/YourNameSchool.txt b/YourNameSchool.txt
new file mode 100644
--- /dev/null	
+++ b/YourNameSchool.txt	
@@ -0,0 +1,1 @@
+МБОУ ВСОШ №2
\ No newline at end of file
Index: schoolSarachik
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/schoolSarachik b/schoolSarachik
new file mode 100644
--- /dev/null	
+++ b/schoolSarachik	
@@ -0,0 +1,1 @@
+МБОУ ВСОШ №2
\ No newline at end of file
Index: listSarachik.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/listSarachik.txt b/listSarachik.txt
new file mode 100644
--- /dev/null	
+++ b/listSarachik.txt	
@@ -0,0 +1,1 @@
+Борисов Дмитрий Алексеевич
\ No newline at end of file
